ddk-parent/pom.xml

```diff
@@ -91,6 +91,7 @@
     <module>../com.avaloq.tools.ddk.xtext.test</module>
     <module>../com.avaloq.tools.ddk.xtext.test.core</module>
     <module>../com.avaloq.tools.ddk.xtext.ui</module>
+    <module>../com.avaloq.tools.ddk.xtext.ui.test</module>

```

Comment:
should also be added to some working set

Reply:
Done.